packages/gatsby/cache-dir/static-entry.js

```diff
@@ -184,53 +184,18 @@ export default (pagePath, callback) => {
     bodyHtml = renderToString(bodyComponent)
   }
 
+  function assetsForCurrentPage() {
+    const isNotRootAndMatchesPage = a =>
+      (a.chunkName === `app` || a.chunkName === page.componentChunkName) &&
+      a.file !== `/` // <- I don't think this is possible
+
+    return {
+      js: stats.assets.js.filter(isNotRootAndMatchesPage),
+      css: stats.assets.css.filter(isNotRootAndMatchesPage),
```

Comment:
i removed the `sort` by preload because it's not always stable :/ so messes up the order